test/e2e/storage/testsuites/provisioning.go

```diff
@@ -279,6 +279,7 @@ func TestDynamicProvisioning(t StorageClassTest, client clientset.Interface, cla
 			// Get entry, get mount options at 6th word, replace brackets with commas
 			command += fmt.Sprintf(" && ( mount | grep 'on /mnt/test' | awk '{print $6}' | sed 's/^(/,/; s/)$/,/' | grep -q ,%s, )", option)
 		}
+		command += " || mount | grep 'on /mnt/test'; false"
```

Comment:
@runcom 

Shouldn't it be like below?

```
command += " || (mount | grep 'on /mnt/test'; false)"
```

Because it works as below:

```
$ true || echo false; false
$ echo $?
1
$ true || (echo false; false)
$ echo $?
0
$ false || (echo false; false)
false
$ echo $?
1
```

Also, if it works well, it would be better to try ```relatime```again.

Reply:
Good point, changing it again